controllers/control/k8ssandratask_controller.go

```diff
@@ -150,7 +150,7 @@ func (r *K8ssandraTaskReconciler) Reconcile(ctx context.Context, req ctrl.Reques
 	if dcs, err := filterDcs(kc, kTask.Spec.Datacenters); err != nil {
 		return r.reportInvalidSpec(ctx, kTask, err.Error())
 	} else {
-		patch := client.MergeFrom(kTask.DeepCopy())
+		patch := client.MergeFromWithOptions(kTask.DeepCopy(), client.MergeFromWithOptimisticLock{})
```

Comment:
This seems to fix `ExecuteSequentialK8ssandraTask`, even though I don't understand exactly how 😕